client/src/pages/Scheduler.js

```diff
@@ -4,15 +4,30 @@ import {
   Paper,
   Grid,
   Divider,
-  Typography,
+  Typography
 } from "@material-ui/core";
 import Overview from "../components/scheduler/Overview";
 import PickDate from "../components/scheduler/PickDate";
 import PickTime from "../components/scheduler/PickTime";
+import { getNextAvailableDate } from "../utils/calendarUtils";
 
 const Scheduler = () => {
   const classes = useStyles();
+
+  //this will be fetched from the server
+  const availTimes={start:"09:00", end:"17:00"}
+  const availDates = [1, 2, 3, 4, 5];
+  //this will be set when picking event type --> pulled from context?
+  const interval="60";
```

Comment:
Should this be based on the duration of the event type eventually?

Reply:
yes, this is just dummy data. Eventually it should pull from event